src/infrastructure/data_reference.py

```
@@ -0,0 +1,162 @@
+class DataReference:
+    source: str
+    country: str
+    parameter: str
+    start_date: datetime
+    end_date: datetime
+    file_path: Path
+    format: str
+    size_mb: float
+    row_count: Optional[int] = None
+
+    def to_dict(self) -> Dict:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)